cmd/golangBuild.go

```diff
@@ -35,6 +35,8 @@ const (
 	golangTestsumPackage        = "gotest.tools/gotestsum@latest"
 	golangCycloneDXPackage      = "github.com/CycloneDX/cyclonedx-gomod/cmd/cyclonedx-gomod@latest"
 	sbomFilename                = "bom.xml"
+	golangciLintCurlURL         = "github.com/golangci/golangci-lint/cmd/golangci-lint"
```

Comment:
Should we change variable name from "golangciLintCurlURL" , since we don't use curl anymore?

Reply:
You’re right. 
Done.